.gitignore

```diff
@@ -134,3 +134,7 @@ dmypy.json
 
 .idea/
 /notebooks/.ipynb_checkpoints/
+/notebooks/ms2deepscore/GNPS-COLLECTIONS-MISC.mgf
+/notebooks/spec2vec/GNPS-COLLECTIONS-MISC.mgf
+/notebooks/spec2vec/matches/
+*test-output.xml
```

Comment:
```suggestion
/notebooks/**
```

Reply:
noo.. why? I think we could just ignore the 
`/notebooks/spec2vec/matches/` and `*test-output.xml`. The rest are needed to run the notebooks and the actual notebooks